CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ Changelog
 - Do not block the UI while wallet is being restored/imported ([PR 457](https://github.com/input-output-hk/daedalus/pull/457))
 - Start and stop Mantis Client from Daedalus main process ([PR 568](https://github.com/input-output-hk/daedalus/pull/568))
 - Add ADA and Mantis logo to loader screens ([PR 584](https://github.com/input-output-hk/daedalus/pull/584))
+- Paper wallets certificate generation ui and api integration ([PR 774](https://github.com/input-output-hk/daedalus/pull/774))
```

Comment:
@tomothespian this should be moved to `vNEXT > Features` section and should be renamed to:
```
Paper wallets certificate generation ([PR 779](https://github.com/input-output-hk/daedalus/pull/779))
```
☝️ mind the PR ID and URL change ;)

Reply:
Fixed